autofill.gypi

```diff
@@ -50,6 +50,7 @@
           '<(libchromiumcontent_dir)/signin_core_browser.lib',
           '<(libchromiumcontent_dir)/signin_core_common.lib',
           '<(libchromiumcontent_dir)/os_crypt.lib',
+          'crypt32.lib',
```

Comment:
I'm definitely not a VS expert and I couldn't find any good docs for it. It looks like the `-l` might actually be ignored, but maybe add for consistency.